scripts/sct_compute_hausdorff_distance.py

```diff
@@ -522,12 +517,13 @@ def main(args=None):
 
         #TODO change back the orientatin of the thinned image
         if param.thinning:
-            sct.run('cp ' + computation.thinning1.thinned_image.file_name + computation.thinning1.thinned_image.ext + ' ../' + sct.extract_fname(input_fname)[1] + '_thinned' + sct.extract_fname(input_fname)[2])
-            if im2_name is not None:
-                sct.run('cp ' + computation.thinning2.thinned_image.file_name + computation.thinning2.thinned_image.ext + ' ../' + sct.extract_fname(input_second_fname)[1] + '_thinned' + sct.extract_fname(input_second_fname)[2])
+            shutil.copy(computation.thinning1.thinned_image.file_name + computation.thinning1.thinned_image.ext,
+                        '../' + sct.extract_fname(input_fname)[1] + '_thinned' + sct.extract_fname(input_fname)[2])
+            if im2_name:
+                shutil.copy(computation.thinning2.thinned_image.file_name + computation.thinning2.thinned_image.ext,
+                            '../' + sct.extract_fname(input_second_fname)[1] + '_thinned' + sct.extract_fname(input_second_fname)[2])
 
-        os.chdir('..')
-        # print 'Total time: ', time.time() - now
```

Comment:
I see these all the time and it "flips my toe nails". I think we should use full path instead of changing dir. It can have dire effect when using python threading. We are not right now, but I can see some situation where it could be valuable.  

(one thread changing dir is changing dir for all the thread in python)